lib/open_telemetry/tracer.ex

```diff
@@ -0,0 +1,45 @@
+defmodule OpenTelemetry.Tracer do
+  @moduledoc """
+
+      use OpenTelemetry.Tracer
+
+      OpenTelemetry.Tracer.with_span "span-1" do
+        ... do something ...
+      end
+
+  """
+
+  defmacro start_span(name,  opts \\ quote(do: %{})) do
+    quote do
+      :ot_tracer.start_span(:opentelemetry.get_tracer(unquote(__MODULE__)), unquote(name), unquote(opts))
```

Comment:
Just to be super clear: `unquote(__MODULE__)` here is `OpenTelemetry.Tracer`. So all of the tracers will default to `OpenTelemetry.Tracer`. Is this what you want?

Reply:
Hah, nope, did not want to unquote that. Thanks!

Reply:
Should be good now.